src/plots/geo/layout_defaults.js

```diff
@@ -46,6 +46,28 @@ function handleGeoDefaults(geoLayoutIn, geoLayoutOut, coerce, opts) {
     var isClipped = geoLayoutOut._isClipped = !!constants.lonaxisSpan[projType];
 
     var visible = coerce('visible');
+    if(visible === false) {
```

Comment:
I think this is still incorrect when the `visible=false` came from the template (which we still need a test for - showing that all the `show*` end up `true` in that case). I guess the condition here should be `if(geoLayoutIn.visible === false)`

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/4483/commits/639913e2f7e77e93a4cd0db40a1a19f67d2b82e4.